src/platform/site-wide/announcements/components/VAMCWelcomeModal.jsx

```diff
@@ -6,44 +6,42 @@ export const VAMC_PATHS = {
   PITTSBURGH: /^\/pittsburgh-health-care\/(.)*$/,
 };
 
-export default class VAMCWelcomeModal extends React.Component {
-  static propTypes = {
-    announcement: PropTypes.shape({
-      region: PropTypes.string.isRequired,
-    }).isRequired,
-  };
+function VAMCWelcomeModal({ dismiss, announcement }) {
```

Comment:
isn't it better to add the `export default` here or fix the next file to add it at the bottom for consistency

Reply:
This one has `propTypes`, so in order to preserve that I think I have to add them after the function definition.  I kinda assumed that once a function had been exported you couldn't add propTypes, but I could be wrong.